src/jdk.incubator.vector/share/classes/jdk/incubator/vector/VectorOperators.java

```diff
@@ -521,9 +521,9 @@ static boolean opKind(Operator op, int bit) {
     /** Produce {@code a>>>(n&(ESIZE*8-1))}.  Integral only. */
     public static final /*bitwise*/ Binary LSHR = binary("LSHR", ">>>", VectorSupport.VECTOR_OP_URSHIFT, VO_SHIFT);
     /** Produce {@code rotateLeft(a,n)}.  Integral only. */
-    public static final /*bitwise*/ Binary ROL = binary("ROL", "rotateLeft", -1 /*VectorSupport.VECTOR_OP_LROTATE*/, VO_SHIFT | VO_SPECIAL);
+    public static final /*bitwise*/ Binary ROL = binary("ROL", "rotateLeft", VectorSupport.VECTOR_OP_LROTATE, VO_SHIFT | VO_SPECIAL);
```

Comment:
I think we can remove the `VO_SPECIAL` flag on `ROL` and `ROR` now it is uniformly managed?